ibridges/utils/path.py

```diff
@@ -82,23 +115,11 @@ def dataobject_exists(self) -> bool:
         """
         return self.session.irods_session.data_objects.exists(str(self))
 
-    def absolute(self):
-        """
-        Return the path if the path starts with '/zone/home', otherwise 
-        concatenate the '/zone/home' prefix to the current path.
-        """
-
     def exists(self) -> bool:
         """
         Check if the path already exists on the iRODS server
         """
-
-    def home(self):
-        """
-        If the session environment defines an 'irods_home', checks if this path exists 
-        and returns the path.
-        If 'irods_home' is not defined, returns the path /zone/home
-        """
+        return self.dataobject_exists() or self.collection_exists()
 
     def walk(self, depth: int):
```

Comment:
The function is not implemented and does nothing. Can we throw a warning?

Reply:
I'm going to throw an error instead if you don't mind!